src/model/simulator.py

```diff
@@ -62,6 +63,7 @@ def simulate(model, method, objective_id, objective_direction, tmy_objectives, t
                     add_moma(model, solution=reference, linear=(method == 'lmoma'))
                     solution = model.optimize()
         except OptimizationError:
+            logger.info(f"Infeasible model")
```

Comment:
There are other reasons that could lead to an optimization error despite infeasibility. So might want to:
```python
except OptimizationError as err:
    logger.info(str(err))
```